actions/class.PropertiesAuthoring.php

```diff
@@ -304,12 +304,52 @@ public function getClassForm(core_kernel_classes_Class $class): tao_helpers_form
                 //save all properties values
                 if (isset($data['properties'])) {
                     $this->saveProperties($data);
+                    $this->populateSubmittedProperties($myForm, $data);
                 }
             }
         }
         return $myForm;
     }
 
+    private function populateSubmittedProperties($myForm, $data): void
+    {
+        if (empty($data['properties'])) {
+            return;
+        }
+        $elementRangeArray = [];
+        $groups = $myForm->getGroups();
+        
+        foreach ($data['properties'] as $prop) {
+            if (empty($prop['range'])) {
+                continue;
+            }
+
+            if (empty($prop['range']) && empty($prop['uri'])) {
+                continue;
+            }
```

Comment:
@gabrielfs7 @chinnu-focaloid @boajer wdyt?
```suggestion
            if (empty($prop['range']) || empty($prop['uri'])) {
                continue;
            }
```

Reply:
I resolved this.  Please check